lib/active_merchant/billing/gateways/authorize_net.rb

```diff
@@ -184,18 +184,8 @@ def scrub(transcript)
       end
 
       def supports_network_tokenization?
-        card = Billing::NetworkTokenizationCreditCard.new({
-          :number => "4111111111111111",
-          :month => 12,
-          :year => 20,
-          :first_name => 'John',
-          :last_name => 'Smith',
-          :brand => 'visa',
-          :payment_cryptogram => 'EHuWW9PiBkWvqE5juRwDzAUFBAk='
-        })
-
         request = post_data(:authorize) do |xml|
-          add_auth_purchase(xml, "authOnlyTransaction", 1, card, {})
+          add_auth_purchase(xml, "authOnlyTransaction", 1, Billing::NetworkTokenizationCreditCard.test_credit_card, {})
```

Comment:
nit: `NetworkTokenizationCreditCard` should be enough. `Billing` module is already implied and will be checked first due to resolution rules and nesting.